locations/fblipke.yml

```diff
@@ -0,0 +1,75 @@
+---
+location: fblipke
+location_nice: FBLipke - Freifunk Berlin
+latitude: 52.484948320
+longitude: 13.443380903
+altitude: 42
+height: 13
+contact_name: 'Guido L.'
+contact_nickname: 'LipkeGu'
+contacts:
+  - 'contact@fblipke.de'
+hosts:
+  - hostname: fblipke-core
+    role: corerouter
+    model: "x86_64"
+    wireless_profile: freifunk_default
+    low_flash: true
+  - hostname: fblipke-ap01
+    role: ap
+    model: "ubnt_nanostation_m2_xw"
+    wireless_profile: freifunk_default
```

Comment:
this is the default for APs and therefore can be removed from the configuration.